torch/optim/adamw.py

```diff
@@ -318,92 +318,6 @@ def step(self, closure=None):
 )
 
 
-def adamw(
```

Comment:
wait is this deletion intended?

Reply:
I merged main branch to this, and [0f02e0a](https://github.com/pytorch/pytorch/commit/0f02e0aa39b3dff15a8e2d1a81d049e838d7c696) moved the function to the end of the file.